### DISCUSSION TITLE
人類是唯一有肥胖問題的靈長類

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1542] [General]
我自己在動物園看猴子猩猩的時候發現他們都很精瘦，很少出現那種超級大胖子，後來研究了一下發現對猩猩猴子這些靈長類同胞來說﻿肥胖 從來不是一個值得困擾的問題－﻿
野生猿猴平均體脂只有 10%，即便被豢養﻿
整天努力把自己吃胖，最多也不超過 25%﻿

﻿
更過分的是，這些猩猩猴子運動量也不高﻿
相較都市人每天建議要走上 8 公里才健康﻿
一頭母黑猩猩只要很 chill 的走一兩公里﻿
就可以擁有體脂 5% 的健美級身材﻿
﻿

﻿
透過化石證據，科學家猜測人類變胖的契機﻿
可能跟智人在演化時腦容量加大三倍有關﻿
－為了穩定供應這顆大大腦所需要的能量
人體於是追加一個更容易囤積脂肪的設定﻿
﻿
此外，因為肌肉會消耗更多卡路里﻿
人體還順手砍掉了很多要分給肌肉的預算﻿
可就算肌肉量更少了，人類每日基礎代謝量﻿
還是比野生猿猴高出近一碗大魯 (500大卡)﻿
可見這顆大大腦吃得有多兇﻿
﻿
但我覺得這種困擾現代人的脂肪儲存機制﻿
或許正是人類能戰勝其他物種的重要武器－﻿

**Comment by poweichen00** [comment]
肥胖在野生猿猴和人類之間的差異確實存在，這差異可能與演化過程和生活方式的改變有關。野生猿猴通常具有較低的體脂百分比，這可能與它們的自然飲食、運動量和基因組等。

**Comment by larry40305** [comment]
現代社會中的過度飲食和久坐生活方式導致了肥胖和健康問題的增加。重要的是要意識到自己的飲食習慣和生活方式對健康的影響，並努力維持均衡的飲食和適度的運動。每個人的身體和代謝情況是獨特的，因此建議諮詢專業醫生或營養師以獲得個人化的建議。
